indico/modules/events/contributions/fields.py

```diff
@@ -52,6 +52,8 @@ def _get_person_link(self, data):
         return super(ContributionPersonLinkListField, self)._get_person_link(data, extra_data)
 
     def _serialize_person_link(self, principal, extra_data=None):
+        if principal.id is None:
+            return super(ContributionPersonLinkListField, self)._serialize_principal(principal.user)
```

Comment:
Actually this will never be called in this case since it is handled in PersonLinkListFieldBase._serialize_principal()